front/components/spaces/SpaceActionsList.tsx

```diff
@@ -100,14 +100,17 @@ export const SpaceActionsList = ({
         id: "actions",
         cell: (info: CellContext<RowData, string>) => (
           <DataTable.MoreButton
-            menuItems={[
-              {
-                label: "Remove tools from space",
-                onClick: async () => onRemoveServer(info.row.original.id),
-                kind: "item",
-                disabled: !isAdmin && space.kind === "global",
-              },
-            ]}
+            menuItems={
+              isAdmin
+                ? [
+                    {
+                      label: "Remove tools from space",
+                      onClick: async () => onRemoveServer(info.row.original.id),
+                      kind: "item",
+                    },
+                  ]
+                : []
```

Comment:
Is the moreButton hidden if there's no item ? (I'm pretty sure it is, but just wanna confirm)
Otherwise we could just hide the button completely instead of having a button with zero actions ?

Reply:
yes, it's hidden automatically if array is empty